### DISCUSSION TITLE
Installing Ultimate Hosts Blacklist

### DISCUSSION THREAD

**Discussion by subsofdan** [Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist#705] [Q&A]
Hey, dear Github Members, 

I'd like to 'install' UHB on my Macbook M1 Max but to be honest, I don't understand anything of what I should do... 

I was searching for a handout/guide for installing host files on Mac (as I understand is that what I should do) but I can't find anything that's understandable for me. 

Could you tell me where I can find a guide for installing it on my macbook? I really have no idea what I should do...

Thank you for your help, I truly appreciate it!

🙏🏻

Daniel

**Comment by JonnyTech** [comment]
Not at a computer, but from memory:

Download https://hosts.ubuntu101.co.za/hosts to your desktop
Open the terminal app
Type the following:
```
sudo cp ~/Desktop/hosts /etc/hosts
```
You will need to enter your password which will not appear as you type

**Reply by subsofdan** [reply]
Thank you for your answer, JonnyTech! ✨🙏🏻✨

**Comment by smed79** [comment]
Use the DNS servers 

- https://github.com/Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist?tab=readme-ov-file#dns-server

Change DNS settings on Mac 

- https://support.apple.com/guide/mac-help/change-dns-settings-on-mac-mh14127/mac
